Packs/ApiModules/Scripts/HTTPFeedApiModule/HTTPFeedApiModule.py

```diff
@@ -445,5 +453,5 @@ def feed_main(feed_name, params=None, prefix=''):
             readable_output, outputs, raw_response = commands[command](client, args)
             return_outputs(readable_output, outputs, raw_response)
     except Exception as e:
-        err_msg = f'Error in {feed_name} integration [{e}]\nTrace\n:{traceback.format_exc()}'
-        return_error(err_msg)
+        err_msg = f'Error in {feed_name} integration [{e}]'
```

Comment:
```suggestion
        err_msg = f'Error in {feed_name} integration:\n [{e}]'
```

Reply:
it will not have any effect on the test module